### PR TITLE
ceph.spec.in: clarify two important comments

### PR DESCRIPTION
**Pull request by smithfarm**
First, the terms "common" and "specific" are vague. Second,
"common" can easily be confused with the ceph-common subpackage.

Fix this by rephrasing to "distro-unconditional dependencies" and
"distro-conditional dependencies", respectively.

Third, move the "distro-unconditional dependencies" header so it
is above the part where these dependencies are actually defined.

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: ceph.spec.in [hunk 1]
```diff
@@ -33,6 +30,9 @@ Source0:	http://ceph.com/download/%{name}-%{version}.tar.bz2
 %if 0%{?fedora} || 0%{?rhel}
 Patch0:		init-ceph.in-fedora.patch
 %endif
+#################################################################################
+# distro-unconditional dependencies
```
Comment (ktdreyer) [inline ceph.spec.in]: maybe reword unconditional -> "independent" ?

Comment (smithfarm) [inline ceph.spec.in]: "distro-independent dependencies" . . . dunno, sounds rather confusing. 

Originally I had written "dependencies that apply across all distros" and "dependencies that are specific only to a particular distro" but that seemed too verbose. Another option would be "non-distro-specific dependencies" and "distro-specific dependencies".

Comment (ktdreyer) [inline ceph.spec.in]: "dependencies that apply across all distro families" sounds fine to me (not too verbose)

**Comment by smithfarm** [comment]
@ktdreyer updated
